daemon/cc_mode/c_service.proto

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is part of trust|me
+ * Copyright(c) 2013 - 2021 Fraunhofer AISEC
+ * Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms and conditions of the GNU General Public License,
+ * version 2 (GPL 2), as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GPL 2 license for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, see <http://www.gnu.org/licenses/>
+ *
+ * The full GNU General Public License is included in this distribution in
+ * the file called "COPYING".
+ *
+ * Contact Information:
+ * Fraunhofer AISEC <trustme@aisec.fraunhofer.de>
+ */
+
+syntax = "proto2";
+
+option java_package = "de.fraunhofer.aisec.trustme.service";
+
+import "container.proto";
+import "common/audit.proto";
+
+message CmldToServiceMessage {
+	reserved 6 to 12, 15;
+	enum Code {
+		reserved 2 to 6, 15, 16;
```

Comment:
Comment to make it clear that the reserved values are those that are available to users of trustm3 built in non-CC-mode?

Reply:
done